src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        book.setTitle("Effective Java");
+        book.setPrice(new BigDecimal("45.99"));
+
+        bookDao.create(book);
+        System.out.println("Created book: " + book);
+
+        Book fetchedBook = bookDao.findById(book.getId()).orElseThrow();
```

Comment:
Using `orElseThrow()` without specifying an exception can lead to a `NoSuchElementException` if the book is not found. Consider providing a custom exception message or handling the absence of the book more gracefully.